packages/esm-patient-registration-app/src/patient-registration/field/id/id-field.test.tsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { Identifiers } from './id-field.component';
+import { Resources, ResourcesContext } from '../../../offline.resources';
+import { Form, Formik } from 'formik';
+import { PatientRegistrationContext } from '../../patient-registration-context';
+import { useConfig } from '@openmrs/esm-framework';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  ...jest.requireActual('@openmrs/esm-framework'),
+  useConfig: jest.fn(),
+}));
+
+describe('Identifiers', () => {
+  const mockedIdentifierTypes = [
```

Comment:
You can use [\_\_mocks\_\_ directory](https://github.com/openmrs/openmrs-esm-patient-management/tree/main/__mocks__)  to store these kind of mock objects.